### PR TITLE
No self-mentioning

### PR DESCRIPTION
**Pull request by VishramKidPG123**

### CODE REVIEW

**Comment by NatalieCrazy** [comment]
patch-21 :typing:

File: main.py [hunk 1]
```diff
@@ -2881,7 +2881,7 @@ async def on_submit(self, interaction: discord.Interaction):
     if not view:
         await message.response.send_message(embed=embed)
     else:
-        await message.response.send_message(person2.mention, embed=embed, view=view)
+        await message.response.send_message(person2.mention if person2!=person1 else None, embed=embed, view=view)
```
Comment (milenakos) [inline main.py:2884]: spacing

**Comment by uku1928** [comment]
it just feels funnier to let catbot ping self
and more consistent

**Comment by milenakos** [comment]
this is not what the pr is about
its about pinging the user if they trade themselves (aka introvert ach)

**Comment by milenakos** [comment]
but yeah selfping is funnyy so im closing

**Pull request closed without merging**